src/components/Board.tsx

```diff
@@ -1,33 +1,119 @@
 import * as React from 'react';
-import { Piece } from '../types';
 import { Layer, Stage } from 'react-konva';
+import { MatchInfo, GameSpec, Piece } from '../types';
+import CanvasImage from './CanvasImage';
+import BoardPiece from './BoardPiece';
+import { AppBar, FlatButton } from 'material-ui';
 
 interface Props {
   pieces: Piece[];
+  matchInfo: MatchInfo;
+  gameSpec: GameSpec;
+  myUserId: string;
+  onReset: () => void;
 }
 
+// for testing purposes
+const testProps: Props = {
```

Comment:
As discussed, let's move this later to a single json which everyone can use for their testing purposes.